src/libcore/str/mod.rs

```diff
@@ -3986,7 +3986,7 @@ impl str {
     /// Note: only extended grapheme codepoints that begin the string will be
     /// escaped.
     ///
-    /// [`char::escape_debug`]: ../std/primitive.char.html#method.escape_debug
+    /// [`char::escape_debug`]: ../../std/primitive.char.html#method.escape_debug
```

Comment:
Wouldn't this also work without the `std` segment? That way, clicking the links in libcore would stay in libcore.

Reply:
All links (at least in this file) are in the form `../../std/` so modifying that is out of scope for this PR :)

Reply:
Okay, must be intentional then